doc/rst/tools/mason/mason.rst

```diff
@@ -13,17 +13,38 @@ that be to create a library to contribute to the community, use an
 existing package, or just to help with the process of building your
 Chapel programs.
 
-.. note::
-
-   **Pardon Our Dust**
-
-   This mason guide is a work-in-progress as mason development itself
-   is still a work-in-progress.  If you're not already using the `main
-   branch <https://chapel-lang.org/docs/main/mason-packages/>`_ of the
-   documentation (which tracks real-time GitHub development), you may
-   find that it contains a more complete set of sections.  We welcome
-   feedback and requests on the guide as we go.
-
+When to Leverage Mason for Chapel Builds
+----------------------------------------
+
+Mason simplifies the build process for Chapel applications, but it's
+not currently a one-size-fits-all solution. Here's a breakdown of
```

Comment:
I recommend deleting this last phrase, "but it's not currently ...".  The next sentence covers the key point that there are possible alternatives.  This phrase might come off as defensive.